sdks/python/apache_beam/coders/coders.py

```diff
@@ -453,6 +454,40 @@ def value_coder(self):
     return self
 
 
+class ProtoCoder(FastCoder):
+  """A Coder for Google Protocol Buffers.
+
+  It supports both Protocol Buffers syntax versions 2 and 3. However,
+  the runtime version of the python protobuf library must exactly match the
+  version of the protoc compiler what was used to generate the protobuf
```

Comment:
yes, right now protobuf is not maintaining backward compatibility even across minor versions. I hope that it doesn't change too often but if that ends up being a problem, we could make a pypi package of all generated files for each version and depend on it rather than checking in the files.